packages/admin-ui/src/pagination/pagination.tsx

```diff
@@ -29,7 +29,7 @@ export const Pagination = createComponent<'div', PaginationOptions>((props) => {
 
   const formatMessage = useMessageFormatter(messages)
 
-  const hasOnlyOnePage = numberOfPages === 1
+  const hasOnlyOnePage = numberOfPages <= 1
```

Comment:
is `hasOnlyOnePage` still a good name for this case? maybe:
```suggestion
 const currentPageLabel =  numberOfPages <= 1
    ? total
    : `${firstPosition} — ${lastPosition}`
```

Reply:
that's true! Good catch